.github/workflows/cd.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot CD to GCP
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-dependabot-merge-action-app:
+    name: Build & Deploy dependabot-merge-action-app
+    runs-on: ubuntu-latest
+
+    env:
+      PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
+      CLOUDRUN_SERVICE_NAME: dependabot-merge-action-app
```

Comment:
let's make this a secret as well so it's configurable

Reply:
Configured as a secret.